### ISSUE TITLE
foundry debug: unknown address at

### ISSUE THREAD

**Issue by chen4903** [foundry-rs/foundry#5410]
### Component

Forge

### Have you ensured that all of these are up to date?

- [X] Foundry
- [X] Foundryup

### What version of Foundry are you on?

forge 0.2.0 (56adbe4 2023-07-16T07:03:29.896803000Z)

### What command(s) is the bug in?

forge debug --debug

### Operating System

Windows

### Describe the bug

I use debug in foundry, but it doesn't show the contract code. I have read these two issues, but doesn't work. [issues#2348](https://github.com/foundry-rs/foundry/issues/2348) and [issues#2947](https://github.com/foundry-rs/foundry/issues/2947)

    2023/07/16  23:09    <DIR>          .
    2023/07/16  21:15    <DIR>          ..
    2023/07/16  23:09                33 .gas-snapshot
    2023/07/16  21:15    <DIR>          .github
    2023/07/16  21:15               156 .gitignore
    2023/07/16  21:24               235 .gitmodules
    2023/07/16  22:49    <DIR>          broadcast
    2023/07/16  22:49    <DIR>          cache
    2023/07/16  21:58               590 foundry.toml
    2023/07/16  21:24    <DIR>          lib
    2023/07/16  21:46    <DIR>          out
    2023/07/16  21:25               140 remappings.txt
    2023/07/16  21:15    <DIR>          script
    2023/07/16  21:15    <DIR>          src
    2023/07/16  21:15    <DIR>          test

input: `forge debug --debug src/Counter.sol --sig "setMember()"`

contract:

    // SPDX-License-Identifier: UNLICENSED
    pragma solidity ^0.8.13;

    contract EvmStudy{

        uint256 public member;
        constructor(){}

        function setMember() public {
            member = 2;
        }
    }

**Comment by madlabman** [comment]
I have the same problem on Linux either on the latest version either at the mentioned 56adbe4 commit, although on 87bc53f it worked. So we can do a binary search, but on my laptop it takes to long to build `foundry`.

**Comment by ZaneH** [comment]
```
OS: macOS 12.2.1 21D62 x86_64
Shell: zsh 5.8
CPU: Intel i7-9700F (8) @ 3.00GHz
```

I've been experiencing the same thing. I've only used `forge test --debug ...`. I also just ran `foundryup` to get the latest updates and it still doesn't see the contracts while debugging.

Here's the command I'm using right now:

```bash
$ forge test -vvv --mc ReenterTest --debug "testAttack()"
```

---

![SCR-20230831-phgi](https://github.com/foundry-rs/foundry/assets/8400251/a78323c9-ad58-4632-adce-94258b20d59a)
